src/Router.js

```diff
@@ -14,6 +17,9 @@ const Router = () => {
         <Route path="/" element={<Main />} />
         <Route path="/product" element={<Product />} />
         <Route path="/detail" element={<Detail />} />
+        <Route path="/tea-set" element={<TeaSet />} />
+        <Route path="/premium-tea" element={<PremiumTea />} />
+        <Route path="/malcha" element={<Malcha />} />
```

Comment:
테스트용으로 라우팅 걸어두시고 추후 삭제되는 페이지인가요?? 👀

Reply:
> 테스트용으로 라우팅 걸어두시고 추후 삭제되는 페이지인가요?? 👀

네!

Reply:
```suggestion
        <Route path="/tea-set" element={<TeaSet />} />
        <Route path="/premium-tea" element={<PremiumTea />} />
        <Route path="/malcha" element={<Malcha />} />
```
@LimJeeIn  
이부분만 삭제해 주세요! 

추가적인 detail은 다음 PR 에서 작업할게요!